src/lib/pages/workflow-query.svelte

```diff
@@ -25,86 +29,127 @@
   };
 
   let queryType: string;
-  let isLoading = false;
+  let initialQueryType: string;
+  let input = '';
+  let initialInput = '';
+  let loading = false;
+  let jsonFormatting = true;
+
+  $: edited = initialQueryType !== queryType || input !== initialInput;
 
   let queryTypes = getQueryTypes({
     namespace,
     workflow: params,
   }).then((queryTypes) => {
     queryType = queryType || queryTypes[0];
+    query(queryType);
```

Comment:
> The only thing I might change is not fetching the initial query by default on page load and instead you need to click "Run Query" always

@Alex-Tideman we can definitely remove this initial query. Wondering if we want a different empty/initial state though ⬇️ 

![Screenshot 2024-10-29 at 11 26 02 AM](https://github.com/user-attachments/assets/fc0484cd-e1e1-4872-9da2-4a87db78e6e4)